### PR TITLE
fix(FixedFooterLayout): remove next js errors when using SSR

### PR DESCRIPTION
**Pull request by marcoskolodny**

### CODE REVIEW

File: src/hooks.tsx [hunk 1]
```diff
@@ -173,7 +173,9 @@ export const useScreenHeight = (): number => {
     return screenHeight;
 };
 
-export const useIsWithinIFrame = (): boolean => window?.top !== window?.self;
+export const useIsWithinIFrame = (): boolean => {
+    return typeof window !== 'undefined' && window.top !== window.self;
```
Comment (atabel) [inline src/hooks.tsx:177]: why don't use `!isServerSide()` here too?

Comment (marcoskolodny) [reply src/hooks.tsx]: fixed

File: src/utils/environment.tsx [hunk 2]
```diff
@@ -1 +1,3 @@
 export const isServerSide = (): boolean => typeof window === 'undefined';
+
+export const isClientSide = (): boolean => typeof window !== 'undefined';
```
Comment (Marcosld) [inline src/utils/environment.tsx:3]: matter of taste: export const isClientSide = (): boolean => !isServerSide();

**Review by Marcosld — APPROVED** [review]

**Pull request merged**